### ISSUE TITLE
setting`colorScheme` on `<FormStepper>` works but throws an error.

### ISSUE THREAD

**Issue by simonpeters** [saas-js/saas-ui#139]
The props of `FormStepper` allow for `colorScheme` to be assigned. This actually works and changes the color of the step indicators. But this also throws an error.

```
client.js?4d30:1 Warning: React does not recognize the `colorScheme` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `colorscheme` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
```

I have the feeling that the `colorScheme` is getting passed to a child component that doens't have the colorScheme prop.

**Comment by Pagebakers** [comment]
Thanks @simonpeters, seems like it is passed down to the dom element indeed. Will get this sorted in 2.0

**Comment by Pagebakers** [comment]
Fixed in 2.0 beta
